cmd/tetragon/flags.go

```diff
@@ -114,6 +115,7 @@ func readAndSetFlags() {
 	option.Config.DataCacheSize = viper.GetInt(keyDataCacheSize)
 
 	option.Config.MetricsServer = viper.GetString(keyMetricsServer)
+	option.Config.MetricsLabelFilter = viper.GetStringMap(keyMetricsLabelFilter)
```

Comment:
Instead of a helper function, I think instead we would be able to introduce changes to the template:
```
{{- if .Values.tetragon.prometheus.enabled }}
  metrics-label-filter: -|
    {{- range .Values.tetragon.prometheus.metricsLabelFilter }}
    - {{ . | quote }}
    {{- end }}
{{- end }}
```
This allows us to parse correctly with `viper.GetStringMap(...)`.
Do you have any concerns with this?

Reply:
@nap32 Will it work with an empty list? Also, how would you pass the list as a command-line flag? Tetragon supports bare metal installations, so this option should be usable outside of the Helm chart too. As long as these two cases are good, changing the format is fine I think.

Reply:
I've tested and don't believe it would work with command line.
I've switched it to your suggestion, thanks!